getconf/base.py

```diff
@@ -169,6 +170,19 @@ def gettimedelta(self, key, default='0d', doc=''):
         else:
             return datetime.timedelta(**{unit: value})
 
+    def getpath(self, key, default=Path(), doc=''):
+        assert (
+            default is None or isinstance(default, Path)
```

Comment:
```suggestion
            default is None or isinstance(default, (Path, str))
```

That would be more forgiving.

Reply:
You lost me, the whole point was to avoid confusion between str and path.

Reply:
The contract of `getpath()` says that it returns a `Path` object. Allowing both strings and Path objects as the default value seems to me like a gesture to the user. As in: *Be conservative in what you do, be liberal in what you accept from others.*